zerver/lib/actions.py

```diff
@@ -1691,7 +1682,7 @@ def do_send_messages(messages_maybe_none: Sequence[Optional[MutableMapping[str,
     # returned.  In practice, this shouldn't matter, as we only
     # mirror single zephyr messages at a time and don't otherwise
     # intermingle sending zephyr messages with other messages.
-    return already_sent_ids + [message['message'].id for message in messages]
```

Comment:
The above comment is no longer accurate and can be deleted with this commit; I'll take care of that.

Reply:
Yes, missed that.